index.js

```diff
@@ -750,6 +750,31 @@
   }
   S.pipe = def('pipe', {}, [$.Array($.Function), a, b], pipe);
 
+  //# on :: (b -> b -> c) -> (a -> b) -> a -> a -> c
+  //.
+  //. Takes a binary function, a unary function, and two values. Returns the
+  //. result of applying the unary function to each value, then passing the
+  //. results through the binary function.
```

Comment:
"Passing through" doesn't sound right to me for some reason. Combinators are sometimes best explained by their definitions. We could write something along these lines:

_Takes a binary function `f`, a unary function `g`, and two values `x` and `y`. Returns `f(g(x))(g(y))`._